Jenkins:
```
10:26:49.458 [ERROR] Failed to execute goal org.eclipse.tycho.extras:tycho-p2-extras-plugin:3.0.0:compare-version-with-baselines (compare-attached-artifacts-with-release) on project org.eclipse.ui.console: Only qualifier changed for (org.eclipse.ui.console/3.11.300.v20221005-1008). Expected to have bigger x.y.z than what is available in baseline (3.11.300.v20220811-1921) -> [Help 1]
```

Should I bump the version? To what?

Thanks,
M.

--- a/org.eclipse.ui.console/META-INF/MANIFEST.MF
+++ b/org.eclipse.ui.console/META-INF/MANIFEST.MF
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.ui.console; singleton:=true
-Bundle-Version: 3.11.300.qualifier
+Bundle-Version: 3.11.400.qualifier
 Bundle-Activator: org.eclipse.ui.console.ConsolePlugin
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin

